`AssertionError: 'ip route 10.0.0.0/8 192.0.2.140 bfd multi-hop source 192.0.2.10profilevyos-test`

Your smoketests contain an error in writing. I will add a fix for this to the PR and also add the IPv6 testcase

--- a/smoketest/scripts/cli/test_protocols_static.py
+++ b/smoketest/scripts/cli/test_protocols_static.py
@@ -69,6 +69,8 @@ routes = {
             '2001:db8::1' : { 'distance' : '10' },
             '2001:db8::2' : { 'distance' : '20', 'interface' : 'eth0' },
             '2001:db8::3' : { 'distance' : '30', 'disable' : '' },
+            '2001:db8::4' : { 'bfd' : '' },
+            '2001:db8::5' : { 'bfd_source' : '2001:db8::ffff' },
         },
         'interface' : {
             'eth0'  : { 'distance' : '40', 'vrf' : 'black' },

